tests/gravity_model_cartesian.prm

```diff
@@ -0,0 +1,100 @@
+# This setup is a copy of the
+# benchmarks/onset-of-convection/convection-box-base.prm
+# contributed by Max Rudolph, with the difference that
+# parameter values are specified explicitly in the input file
+# (rather than through an ipython notebook).
```

Comment:
I dont think the detailed reference to the benchmark cookbook here is necessary. You have modified this file quite a bit and it serves a different purpose. It would be more useful to describe the purpose of this test, e.g. something like:
```
This is a test based on benchmarks/onset-of-convection/convection-box-base.prm. It makes sure that the gravity plugin 'function' works correctly for a  Cartesian coordinate system.
```